Makefile

```diff
@@ -23,8 +23,25 @@ DOCKER_RUN := mkdir -p ../.go-pkg-cache bin $(GOMOD_CACHE) && \
 MAKE_DIRS=$(shell ls -d */)
 
 generate:
-	make -C api gen-files 
-	make -C libcalico-go gen-files
+	$(MAKE) -C api gen-files
+	$(MAKE) -C libcalico-go gen-files
+	$(MAKE) -C felix protobuf
+	rm app-policy/proto/*.pb.go
+	$(MAKE) -C app-policy proto
+
+fix:
```

Comment:
I don't think this target is needed - running `make fix` in the top level directory should already work, no?

Reply:
oh heh - you know i didn't even try ..... my bad.

Reply:
oooh it's magic.

sorry bout that